tests/testthat/test-get_available_datasets.R

```diff
@@ -63,3 +63,21 @@ test_that("National level datasets can be filtered for", {
   expect_equal(nrow(dplyr::filter(nat, class %in% "Italy")), 0)
   expect_equal(unique(nat$type), "national")
 })
+
+package_avaliable_data <- get_available_datasets()
```

Comment:
put this inside the test_that environment to make the error more informative if it happens.